tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py36,py37,pre-commit
+envlist = py36,py37,py38,pre-commit
```

Comment:
a rule of thumb for open source, you should never do any of the following unless asked:

- changing configuration files
- changing build infrastructure
- changing gitignore
- etc.

please rebase this out

Reply:
fair enough, I've removed it